ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.11.yaml

```diff
@@ -51,6 +51,72 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: check-aws-no-capabilities
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      BASELINE_CAPABILITY_SET: None
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        IMAGESTREAM_ALLOWED_LIST="cli cli-artifacts driver-toolkit installer installer-artifacts must-gather oauth-proxy tests tools"
+        TEMPLATE_ALLOWED_LIST=""
+        function exists_in_list() {
+          LIST=$1
+          DELIMITER=$2
+          VALUE=$3
+          LIST_WHITESPACES=`echo $LIST | tr "$DELIMITER" " "`
+          for x in $LIST_WHITESPACES; do
+            if [ "$x" = "$VALUE" ]; then
+              return 0
+            fi
+          done
+          return 1
+        }
+
+        function check_present_list_is_correct() {
+          KIND="${1}"
+          shift
+          CURRENT_LIST="${@}"
+          for i in $CURRENT_LIST;
+            do
+              if [ "$KIND" == "imagestreams" ] ; then
+                if exists_in_list "$IMAGESTREAM_ALLOWED_LIST" " " $i; then
+                  echo "imagestream $i is present, which is expected"
+                else
+                  echo "$i imagestream should not be present"
+                  return 1
+                fi
+              elif [ "$KIND" == "templates" ] ; then
+                if exists_in_list "$TEMPLATE_ALLOWED_LIST" " " $i; then
```

Comment:
You can avoid this switching with:

```sh
assert_empty templates "${TEMPLATE_ALLOWED_LIST}" &&
assert_empty imagestreams "${IMAGESTREAM_ALLOWED_LIST}"
```

But :shrug:, it should work fine this way too.

/lgtm